habitat_sim/registry.py

```diff
@@ -109,7 +109,7 @@ def _wrapper(noise_model: Type[SensorNoiseModel]):
 
     @classmethod
     def register_pose_extractor(
-        cls, pose_extractor: Optional[Type] = None, *, name: Optional[str] = None
+        cls, pose_extractor: Optional[Type] = None, name: Optional[str] = None
```

Comment:
Can you add the `*` back?  `name` only works as keyword arg when this method is a decorator and the `*` makes that clear.